src/store/traction/pacbio/runCreate/run.js

```diff
@@ -183,6 +183,23 @@ const hasPlateAttributes = ({ sequencing_kit_box_barcode, wells_attributes }) =>
   return sequencing_kit_box_barcode || wells_attributes.length > 0
 }
 
+/**
+ * @param {run} - A run object
+ * @param {smrtLinkVersion} - The SMRT Link Version of the run
+ * @returns {run} - A run object
+ */
+const SMRT_LINK_VERSION_V12_SEQUEL_IIE = 'v12_sequel_iie'
+
+const createRunPayload = (run, smrtLinkVersion) => {
```

Comment:
Is this to catch edge cases where a v12 smrtlink run has a dna_control_complex_box_barcode value by accident? Just checking because runs are created with a null value for this by default.

Reply:
@BenTopping Yes I just thought if they select v11, add a DNA Box Barcode, then change it to v12. The DNA box barcode would still be in the runCreate store object, and therefore sent to the Service. As the DNA Box Barcode is optional in the Service, this would let the v12 run be created with it. So, I thought to explicitly remove it here to make sure it isn't persisted.

Reply:
Yeah thats fair enough. We do run an action called 'setSmrtLinkVersion' when the smrtlink version is changed. We could add clear the dcbb there depending on the smrtLinkVersion? I dont know if thats a better solution or not though.

Reply:
@BenTopping Ye sure I wondered the same. The mutation that the `setSmrtLinkVersion` calls is very specific to just replacing the stores `smrtLinkVersion`. So in the end I thought the mutation is better of just having that one task to keep it clean, rather than the mutation changing two different objects in the store.